shaders/glsl/color.fragment

```diff
@@ -35,42 +35,42 @@ float rand_bilinear(highp vec2 coord){
 	return result;
 }
 
-//skyplane  fragment shader and maybe clouds
+// Skyplane fragment shader and maybe clouds
 void main()
 {
     
-	float isRain = 1.0 - length(normalize(vec3(1.0)) - normalize(FOG_COLOR.rgb));//calculate how complimentary fog color wiht gray color
+	float isRain = 1.0 - length(normalize(vec3(1.0)) - normalize(FOG_COLOR.rgb));// Calculate how complimentary fog color with gray color
 	isRain = pow(isRain, 16.0);
 	if(FOG_CONTROL.g > 0.95){
 		isRain = 0.0;
 	}
 
-	// float isNight = 1.0 - pow(length(FOG_COLOR.rgb), 0.01);
+	//float isNight = 1.0 - pow(length(FOG_COLOR.rgb), 0.01);

```

Comment:
I'm not sure but I think VS code adds this margins. In this case I think there is no really need to edit it

Reply:
Removing the margin on the commented code differentiates it from the note comments imo

Reply:
Ahh k understandable :-)